frontend/src/components/RegistrationForm.js

```diff
@@ -9,6 +11,34 @@ export default function RegistrationForm() {
 
     async function signUp(event) {
         event.preventDefault();
+
+        try {
+            const response = await axios.post(
+                "http://localhost:8000/api/register",
+                {
+                    name,
+                    email,
+                    password,
```

Comment:
you have confirm password as input but you did not pass it to BE api.
Pass it and make adjustment of your BE api to check confirm password